.vib/concourse/goss/web/vars.yaml

```diff
@@ -0,0 +1,34 @@
+secrets:
+  localUser: "bitnamiUser"
+  localPassword: "ComplicatedPassword123!4"
+web:
+  podSecurityContext:
+    fsGroup: 1002
+  containerSecurityContext:
+    runAsUser: 1002
+  serviceAccount:
+    automountServiceAccountToken: true
+service:
+  web:
+    ports:
+      http: 80
+      https: 444
```

Comment:
You are missing the `http` test checking the https port of the K8s svc.

Reply:
The `https` test is skipped on purpose as the chart currently presents some problems with it. We may add it in the future when it is fixed, though.